app/views/practices/show/authenticated/_show_authenticated.html.erb

```diff
@@ -309,17 +307,17 @@
 <% end %>
 
 <!-- "Support Team" || "Contact" section -->
-<% if @practice.va_employees.any? || @practice.support_network_email.present? %>
-  <section class="margin-bottom-7">
-    <div class="grid-container">
-      <div class="grid-row grid-gap">
-        <div class="desktop:grid-col-3 grid-col-auto z-bottom">&nbsp;</div>
-        <div class="desktop:grid-col-9 grid-col-12 p0">
-          <div class="contact-section practice-section">
-            <%= render partial: 'support_team' %>
-          </div>
-        </div>
-      </div>
-    </div>
-  </section>
-<% end %>
+<%# if @practice.va_employees.any? || @practice.support_network_email.present? %>
```

Comment:
can we delete this code if we're not using it?

Reply:
removed :)